docs/specs/field_edit_markup.md

```diff
@@ -0,0 +1,1295 @@
+    </div>
+</div>
+```
+
+## Map Location `ezgmaplocation`
+
+### Options
+
+* Required
+
+### Markup
+
+```html
+<div class="ez-field-edit ez-field-edit-ezgmaplocation">
+    <div class="ez-field-edit-text-zone">
+        <label class="ez-field-definition-name"><!-- no for attribute -->
+            Field Definition Name
+        </label>
+    </div>
+    <div class="ez-field-edit-ui">
+        <fieldset>
+            <div class="ez-sub-field ez-sub-field-latitude">
+                <div class="ez-sub-field-text-zone">
+                    <label class="ez-sub-field-name" for="auto-generated-id-lat">
+                        Latitude
+                    </label>
+                </div>
+                <div class="ez-sub-field-ui">
+                    <input type="number" step="any" min="-90" max="90" id="auto-generated-id-lat" name="auto-generated-name-lat">
+                </div>
+            </div>
+            <div class="ez-sub-field ez-sub-field-longitude">
+                <div class="ez-sub-field-text-zone">
+                    <label class="ez-sub-field-name" for="auto-generated-id-lon">
+                        Longitude
+                    </label>
+                </div>
+                <div class="ez-sub-field-ui">
+                    <input type="number" step="any" min="-90" max="90" id="auto-generated-id-lon" name="auto-generated-name-lon">
+                </div>
+            </div>
+            <div class="ez-sub-field ez-sub-field-address">
+                <div class="ez-sub-field-text-zone">
+                    <label class="ez-sub-field-name" for="auto-generated-id-address">
+                        Address
+                    </label>
+                </div>
+                <div class="ez-sub-field-ui">
+                    <input type="text" id="auto-generated-id-address" name="auto-generated-name-address">
+                </div>
+            </div>
+        </fieldset>
+    </div>
+</div>
+```
+
+When set required, the latitude and longitude inputs receives the `required`
+attribute.
+
+## Image `ezimage`
+
+### Options
+
+* Required
+* Maximum file size
+
+### Markup
+
+Image Field edit markup is very similar to File Field edit markup, the main
+differences are:
+
+* the file input receives an `accept` attribute with the accepted mimetypes
+* the *edit preview* is different to render the store image when there's one and
+  to allow filling an alternative text.
+
+When the Field is empty, the markup should be:
+
+```html
+<div class="ez-field-edit ez-field-edit-ezbinaryfile">
+    <div class="ez-field-edit-text-zone">
+        <label class="ez-field-definition-name" for="auto-generated-id">
+            Field Definition Name
+        </label>
+    </div>
+    <div class="ez-field-edit-ui">
+        <input type="file" accept="image/jpg, image/gif, image/png" id="auto-generated-id" name="auto-generated-name">
```

Comment:
The problem is not only in adding svg mimetype here. The problem that the Image is not able to handle SVG document.
See also discussion in v1 bug https://jira.ez.no/browse/EZP-25208 and related story: https://jira.ez.no/browse/EZP-25422 (I'm sure there are others)